### PR TITLE
En 5633 code coverage improvements 2

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: process/interceptors/processor/trieNodeInterceptorProcessor_test.go [hunk 1]
```diff
@@ -65,6 +66,43 @@ func TestTrieNodesInterceptorProcessor_SaveShouldPutInCacher(t *testing.T) {
...
+			assert.Fail(t, "should have not arrived here")
+			return false
+		},
+	}
+	tnip, _ := processor.NewTrieNodesInterceptorProcessor(cacheMock)
+
+	intData := interceptedBlocks.InterceptedHeader{}
+	slc := make([]process.InterceptedData, 0)
+	slc = append(slc, &intData)
+	tnip.SignalEndOfProcessing(slc)
```
Comment (AdoAdoAdo) [inline process/interceptors/processor/trieNodeInterceptorProcessor_test.go]: how does this test work, where do you verify that it errors?

File: process/smartContract/hooks/blockChainHook_test.go [hunk 2]
```diff
@@ -4,15 +4,16 @@ import (
 	"bytes"
 	"encoding/hex"
 	"fmt"
-	"github.com/ElrondNetwork/elrond-go/process"
-	"math/big"
-	"testing"
-
+	"github.com/ElrondNetwork/elrond-go/data"
```
Comment (AdoAdoAdo) [inline process/smartContract/hooks/blockChainHook_test.go]: go imports

File: process/interceptors/processor/trieNodeInterceptorProcessor_test.go (hunk 1 above)
Comment (bogdan-rosianu) [reply process/interceptors/processor/trieNodeInterceptorProcessor_test.go]: in PutCalled there is an `assert.Fail()` so it should not arrive there. renamed test

File: process/block/shardblock_test.go [hunk 3]
```diff
@@ -1550,8 +1551,8 @@ func TestShardProcessor_CommitBlockStorageFailsForHeaderShouldErr(t *testing.T)
 	body := make(block.Body, 0)
 	hdrUnit := &mock.StorerStub{
 		GetCalled: func(key []byte) (i []byte, e error) {
-			hdr, _ := marshalizer.Marshal(&block.Header{})
-			return hdr, nil
+			hdrToRet, _ := marshalizer.Marshal(&block.Header{})
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply process/block/shardblock_test.go]: done

File: process/block/shardblock_test.go [hunk 4]
```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/gin-gonic/gin/json"
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply process/block/shardblock_test.go]: done

File: hashing/hasher_test.go [hunk 5]
```diff
@@ -28,12 +28,30 @@ func TestFnv(t *testing.T) {
...
 
+func TestingNilInterface(t *testing.T, h hashing.Hasher) {
+
+	res := h.IsInterfaceNil()
+
+	assert.False(t, res)
+
+}
+
+func TestingSize(t *testing.T, h hashing.Hasher) {
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply hashing/hasher_test.go]: done. renamed the rest of the methods as well

File: display/converters_test.go [hunk 6]
```diff
@@ -35,3 +43,35 @@ func TestDisplayByteSlice_WithSetCanBeCalledConcurrently(t *testing.T) {
...
+
+	hash := []byte("short")
+	hexHash := hex.EncodeToString(hash)
+
+	res := display.ToHexShort(hash)
+
+	assert.Equal(t, hexHash, res)
+}
+
+func TestToHexShort_LongHashShouldTrim(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply display/converters_test.go]: renamed

File: display/converters_test.go [hunk 7]
```diff
@@ -35,3 +43,35 @@ func TestDisplayByteSlice_WithSetCanBeCalledConcurrently(t *testing.T) {
 
 	wg.Wait()
 }
+
+func TestToHexShort_EmptyHashShouldReturnEmptyString(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply display/converters_test.go]: renamed

File: display/converters_test.go [hunk 8]
```diff
@@ -35,3 +43,35 @@ func TestDisplayByteSlice_WithSetCanBeCalledConcurrently(t *testing.T) {
...
+	t.Parallel()
+
+	hash := []byte("")
+	res := display.ToHexShort(hash)
+
+	assert.Equal(t, 0, len(res))
+}
+
+func TestToHexShort_ShortHashShouldDoNothing(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply display/converters_test.go]: renamed

File: process/smartContract/hooks/blockChainHook_test.go (hunk 2 above)
Comment (bogdan-rosianu) [reply process/smartContract/hooks/blockChainHook_test.go]: done

Comment (raduchis) [inline process/smartContract/hooks/blockChainHook_test.go]: format imports

File: hashing/hasher_test.go [hunk 9]
```diff
@@ -28,12 +28,30 @@ func TestFnv(t *testing.T) {
...
+
+	res := h.Size()
+
+	assert.Equal(t, 0, res%2)
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply hashing/hasher_test.go]: done

File: process/smartContract/hooks/blockChainHook_test.go (hunk 2 above)
Comment (bogdan-rosianu) [reply process/smartContract/hooks/blockChainHook_test.go]: done

Comment (raduchis) [reply process/smartContract/hooks/blockChainHook_test.go]: ok

File: hashing/hasher_test.go (hunk 9 above)
[the remark this answers is not in the document]
Comment (raduchis) [reply hashing/hasher_test.go]: ok

**Pull request merged**